proto/cosmos/evm/erc20/v1/tx.proto

```diff
@@ -20,6 +20,11 @@ service Msg {
   rpc ConvertERC20(MsgConvertERC20) returns (MsgConvertERC20Response) {
     option (google.api.http).get = "/cosmos/evm/erc20/v1/tx/convert_erc20";
   };
+  // ConvertCoin mints a ERC20 token representation of the native Cosmos coin
+  // that is registered on the token mapping.
+  rpc ConvertCoin(MsgConvertCoin) returns (MsgConvertCoinResponse) {
+    option (google.api.http).get = "/cosmos/evm/erc20/v1/tx/convert_coin";
```

Comment:
In what cases is this gateway endpoint used?
It's a bit confusing because it's msg_server and not query.

Reply:
It converts a coin from a bank token to an ERC20.